planner/core/task.go

```diff
@@ -1321,6 +1321,10 @@ func (p *basePhysicalAgg) convertAvgForMPP() *PhysicalProjection {
 		Value:   types.NewDatum(0),
 		RetType: ft,
 	}
+	paramOne := &expression.Constant{
```

Comment:
There is a function called "expression.NewOne". Check if it can work.

Reply:
ok, also replace one, zero, null